packages/react-devtools-scheduling-profiler/src/import-worker/preprocessData.js

```diff
@@ -80,6 +85,23 @@ export function getLanesFromTransportDecimalBitmask(
   return lanes;
 }
 
+const laneToLabelMap: Map<number, string> = new Map();
+function updateLaneToLabelMap(laneLabelTuplesString: string): void {
+  // These marks appear multiple times in the data;
+  // We only need to extact them once.
+  if (laneToLabelMap.size === 0) {
+    const laneLabelTuples = laneLabelTuplesString.split(',');
+    for (let laneIndex = 0; laneIndex < laneLabelTuples.length; laneIndex++) {
+      // The numeric lane value (e.g. 64) isn't important.
+      // The profiler parses and stores the lane's position within the bitmap,
+      // (e.g. lane 1 is index 0, lane 16 is index 4).
+      laneToLabelMap.set(laneIndex, laneLabelTuples[laneIndex]);
+    }
+  }
+}
+
+let profilerVersion = null;
+
 function getLastType(stack: $PropertyType<ProcessorState, 'measureStack'>) {
   if (stack.length > 0) {
```

Comment:
wonder why the syntax highlighting seems to have gotten screwed up in this file

Reply:
Probably the awkward Flow syntax `$PropertyType<ProcessorState, 'measureStack'>`